schemas/telemetry/mobile-event/mobile-event.1.parquetmr.txt

```diff
@@ -0,0 +1,45 @@
+message mobile_event {
+  required group metadata {
+    required binary documentId (UTF8);
+    required int64  Timestamp;
+    optional binary Date (UTF8);
+    required binary geoCountry (UTF8);
+    required binary geoCity (UTF8);
+    required binary appBuildId (UTF8);
+    required binary appName (UTF8);
```

Comment:
Please add the following fields to `metadata`:

```
required binary appVersion (UTF8);
required double sampleId;
required binary appUpdateChannel (UTF8); 
```

Reply:
I had to make these `optional` otherwise the tests would fail?

Reply:
Looks like you have to ensure the metadata fields are available here: https://github.com/mozilla-services/mozilla-pipeline-schemas/blob/master/tests/hindsight/run/output/test_parquet_schemas.lua#L60